src/fprime/cookiecutter_templates/cookiecutter-fprime-deployment/{{cookiecutter.deployment_name}}/Top/{{cookiecutter.deployment_name}}Topology.cpp

```diff
@@ -147,8 +147,10 @@ void setupTopology(const TopologyState& state) {
     regCommands();
     // Project-specific component configuration. Function provided above. May be inlined, if desired.
     configureTopology();
+    // Component-specific configurations using fpp phases. Function provided by autocoder.
+    configComponents(state);
```

Comment:
LGTM

@LeStarch can you confirm the order in which these functions are called is correct, and no-ops are generated when no phases/parameters are defined?

Reply:
I independently checked this syntax with a fresh deployment, and built it straight out. No errors. However, Michael please feel free to verify!